inference-engine/tests/functional/plugin/shared/src/behavior/stress_tests.cpp

```diff
@@ -0,0 +1,52 @@
+// Copyright (C) 2019 Intel Corporation
+// SPDX-License-Identifier: Apache-2.0
+//
+
+#include "behavior/stress_tests.hpp"
+#include "ngraph_functions/subgraph_builders.hpp"
+
+namespace LayerTestsDefinitions {
+
+std::string MultipleAllocations::getTestCaseName(const testing::TestParamInfo<MultipleAllocationsParams>& obj) {
+    LayerTestsUtils::TargetDevice targetDevice;
+    unsigned int allocationsCont;
+
+    std::tie(targetDevice, allocationsCont) = obj.param;
+    std::ostringstream result;
+    result << "targetDevice=" << targetDevice << "_";
+    result << "allocationsCont=" << allocationsCont;
+    return result.str();
+}
+
+void MultipleAllocations::SetUp() {
+    std::tie(targetDevice, m_allocationsCount) = this->GetParam();
+    function = ngraph::builder::subgraph::makeSplitConvConcat();
+}
+
+TEST_P(MultipleAllocations, InferWorksCorrectAfterAllocations) {
```

Comment:
There is no one word `nightly` in the functional tests.
@iefode Do I understand correctly that if the test does not have the prefix `smoke`, then it is a `nightly `test?

Reply:
If test has `prefix` substring in name it will be run in `pre-commit`. If doesn't have it will not be run in `pre-commit` - it's the only difference. `nightly_` prefix doesn't have specific meaning right now as far as I know